warehouse/query-core/src/test/java/datawave/query/ExcerptTest.java

```diff
@@ -225,6 +225,68 @@ public void simpleTest() throws Exception {
         runTestQuery(queryString, format.parse("19000101"), format.parse("20240101"), extraParameters, goodResults);
     }
 
+    @Test
+    public void simpleTestBefore() throws Exception {
+        Map<String,String> extraParameters = new HashMap<>();
+        extraParameters.put("include.grouping.context", "true");
+        extraParameters.put("hit.list", "true");
+        extraParameters.put("return.fields", "HIT_EXCERPT");
+        extraParameters.put("query.syntax", "LUCENE");
+
+        String queryString = "QUOTE:(farther) #EXCERPT_FIELDS(QUOTE/2/before)";
+
+        // not sure why the timestamp and delete flag are present
```

Comment:
any idea why the ts and delete flag are showing up in the EXCERPT?

Reply:
Haven't looked into that any further - I can try tracing it down if we think it's necessary before merging, or we can break that into a different ticket to get the functionality added in faster if not.

Reply:
Upon further review - HIT_EXCERPT is added to the document as an Attribute which includes ts and delete flags whereas most (but not all) of the other fields are added as TypeAttributes which do not include ts or delete flags. This has been the case with the ExcerptTransform class since April 2022.

Reply:
Nice hunting